graphql.go

```diff
@@ -184,9 +184,9 @@ func SubscribeResolverTimeout(timeout time.Duration) SchemaOpt {
 
 // Directives defines the implementation for each directive.
 // Per the GraphQL specification, each Field Directive in the schema must have an implementation here.
-func Directives(directives map[string]interface{}) SchemaOpt {
+func Directives(ds ...directives.Directive) SchemaOpt {
```

Comment:
What does the `s` in `ds` stand for? Maybe use just `d`?

Reply:
Was intended be plural, i.e. `d`irective`s`, but in the context `d` would be fine